### ISSUE TITLE
Update broccoli-funnel dependency

### ISSUE THREAD

**Issue by blimmer** [lifegadget/ember-cli-bootstrap-sassy#27]
To `^1.0.0`. I think it should be a clean update. It will fix this warning when `npm install`-ing against a project using 1.0.0+ of funnel:

```
npm WARN unmet dependency /Users/blimmer/code/my-app/node_modules/ember-cli-bootstrap-sassy requires broccoli-funnel@'^0.2.3' but will load
```

**Comment by yankeeinlondon** [comment]
Will get to this over the weekend

**Comment by yankeeinlondon** [comment]
@blimmer i've updated all dependencies but haven't published to `npm` because it's not working yet. No errors but the `index.js`'s `treeForStyles()` function seems to have broken. Here's the code:

```
treeForStyles: function() {
  var bootstrapPath = path.join(this.app.bowerDirectory, 'bootstrap-sass', 'assets/stylesheets');
  var bootstrapTree = new Funnel(this.treeGenerator(bootstrapPath), {
    srcDir: '/',
    destDir: '/app/styles'
  });
  return bootstrapTree;
}
```

Seems like there may have been some breaking changes to either `treeForStyles` or `Funnel`? Do you have any insight into this?

**Comment by blimmer** [comment]
Hmm, I was looking at the [treeFor docs](https://github.com/ember-cli/ember-cli/blob/master/ADDON_HOOKS.md#treefor) and I don't see any recent changes. If you want, do you want to push a branch up that I could play with?

**Comment by yankeeinlondon** [comment]
@blimmer thanks for the offer. I've created a branch called 1.13.13.

**Comment by blimmer** [comment]
Still working on this, but wanted to give a quick update. Checked out before your 1.13.13 commit and **just** upgraded broccoli-funnel and everything looks good. I think it must be something else in that commit that messed something up. Still digging but wanted to give an update.

**Comment by blimmer** [comment]
All fixed up and #31 is available for your review.
